DotNetHowToMultipleDataSources/Hotel.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace AzureSearch.SDKHowTo
+{
+    using System;
+    using Microsoft.Azure.Search;
+    using Microsoft.Azure.Search.Models;
+    using Microsoft.Spatial;
+    using Newtonsoft.Json;
+
+    // The SerializePropertyNamesAsCamelCase attribute is defined in the Azure Search .NET SDK.
+    // It ensures that Pascal-case property names in the model class are mapped to camel-case
+    // field names in the index.
+    [SerializePropertyNamesAsCamelCase]
+    public partial class Hotel
+    {
+        [System.ComponentModel.DataAnnotations.Key]
+        [IsFilterable]
+        public string HotelId { get; set; }
+
+        [IsFilterable, IsSortable, IsFacetable]
+        public double? BaseRate { get; set; }
+
+        [IsSearchable]
```

Comment:
Do you want to use the Microsoft English analyzer here?

Reply:
FYI Mike Carter redid the data for Hotels -- it has 50 hotels, and I think it has different fields.  Your code with the 3 records is good but you might consider updating it to use rows from the dataset at some point.  I believe the plan for this dataset is to host it in Azure SQL DB, same as what we do for the real estate sample data.

Reply:
I'm fine with this sample being self-contained and not using the new Hotels data.